DangerousAreas.md

```diff
@@ -0,0 +1,11 @@
+## Dangerous Areas of Dublin
+
+* Crumlin
```

Comment:
I don't think it's necessary? As there aren't that many areas for each district. And some of the area are in the middle of two districts. Could break them into cities/counties in the future though.